tools/scripts/track-pr

```diff
@@ -95,17 +110,70 @@ def current_project_id() -> str:
 
 
 def cherry_pick_pr(pr_id: str) -> None:
-    commit = gql.get_pr_merge_commit(id=pr_id)["node"]["mergeCommit"]["oid"]
-    print(f"Cherry-picking {commit}")
+    if "determined-ee" in run_capture("git", "remote", "get-url", CLONED_REMOTE):
+        # TODO Add the separate logic for running EE-only cherry-picks.
+        print("Skipping EE PR, marking as conflicted for human attention")
+        add_item_to_project(current_project_id(), pr_id, FIX_CONFLICT_STATUS)
+        return
+
+    pr_commit = gql.get_pr_merge_commit(id=pr_id)["node"]["mergeCommit"]["oid"]
+    print(f"Cherry-picking {pr_commit}")
 
     try:
-        # TODO Actually run Git to do the cherry-pick.
-        status = NEEDS_TESTING_STATUS
+        # Find and fetch the PR commit and both release branches.
+        release_branch = max(
+            line.split()[1]
+            for line in run_capture(
+                "git", "ls-remote", CLONED_REMOTE, "refs/heads/release-*"
+            ).splitlines()
+        )[len("refs/heads/") :]
+        ee_release_branch = f"{release_branch}-ee"
+        print(f"Found release branch {release_branch}")
+
+        run(
+            "git",
+            "fetch",
+            "--depth=2",
+            CLONED_REMOTE,
+            pr_commit,
+            f"{release_branch}:{release_branch}",
+        )
+
+        ee_remote = "ee"
+        run("git", "remote", "add", ee_remote, "https://github.com/determined-ai/determined-ee.git")
+        run("cat", ".git/config")
+        auth = base64.b64encode(f"x-access-token:{GITHUB_TOKEN}".encode()).decode()
+        run(
+            "git",
+            "config",
+            "--local",
+            "http.https://github.com/.extraHeader",
+            f"Authorization: Basic {auth}",
+        )
+        run("git", "fetch", "--depth=2", ee_remote, f"{ee_release_branch}:{ee_release_branch}")
+
+        # Perform both cherry-picks.
+        run("git", "config", "user.email", "automation@determined.ai")
+        run("git", "config", "user.name", "Determined CI")
+        run("git", "checkout", f"{release_branch}")
+        run("git", "cherry-pick", "-x", pr_commit)
+        run("git", "checkout", f"{ee_release_branch}")
```

Comment:
I know I said f-strings are awesome just a few lines up, but they could probably be dropped here. :)

Reply:
Heh, I think these used to be `f"{remote}/{branch}"` and then I changed things around.